src/messages/manifestjson.js

```diff
@@ -107,3 +107,13 @@ export function manifestPropMissing(property) {
 
 export const PROP_NAME_MISSING = manifestPropMissing('name');
 export const PROP_VERSION_MISSING = manifestPropMissing('version');
+
+export const NO_MESSAGES_FILE = {
+  code: 'NO_MESSAGES_FILE',
+  legacyCode: null,
+  message: _('The "default_locale" is missing localisations.'),
```

Comment:
I think the Mozilla style guide says to use American English, at least as the default locale.